packages/host/tests/acceptance/code-submode/file-tree-test.ts

```diff
@@ -592,6 +593,39 @@ module('Acceptance | code submode tests', function (hooks) {
       'expected near-bottom file to be visible after opening it',
     );
   });
+
+  test('can open files in base realm', async function (assert) {
+    let codeModeStateParam = stringify({
+      stacks: [
+        [
+          {
+            id: `${testRealmURL}Person/1`,
+            format: 'isolated',
+          },
+        ],
+      ],
+      submode: 'code',
+      fileView: 'browser',
+      codePath: `http://localhost:4201/base/cards-grid.gts`,
+    })!;
+
+    await visit(
+      `/?operatorModeEnabled=true&operatorModeState=${encodeURIComponent(
+        codeModeStateParam,
+      )}`,
+    );
+
+    await waitFor('[data-test-file="cards-grid.gts"]');
+
+    await click('[data-test-file="cards-grid.gts"]');
+    await waitUntil(
```

Comment:
Does `await waitFor('[data-test-file="cards-grid.gts"].selected')` work?

Reply:
It works. Thanks!